### PR TITLE
feature(cli): add get command

### PR DESCRIPTION
**Pull request by larme**
## Checklist:

- [ ] My code follows the bentoml code style, both `make format` and
  `make lint` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: bentoml/_internal/cli/bento_management.py [hunk 1]
```diff
@@ -52,16 +52,27 @@ def add_bento_management_commands(
...
-        type=click.Choice(["tree", "json", "yaml", "path"]),
-        default="tree",
+        type=click.Choice(["json", "yaml", "path"]),
+        default="yaml",
     )
     def get(bento_tag, output):
         """Print Bento details by providing the bento_tag
 
         bentoml get FraudDetector:latest
         bentoml get FraudDetector:20210709_DE14C9
```
Comment (sauyon) [inline bentoml/_internal/cli/bento_management.py:62]: Maybe add some output examples?

File: bentoml/_internal/cli/model_management.py [hunk 2]
```diff
@@ -54,16 +54,27 @@ def model_cli():
...
 
         bentoml model get FraudDetector:latest
         bentoml model get FraudDetector:20210709_DE14C9
         """
-        pass
+        res = model_store.get(model_tag).info
+        console = Console()
+
+        if output == "path":
+            path = model_store._fs.getsyspath(res.tag.path())
```
Comment (sauyon) [inline bentoml/_internal/cli/model_management.py:70]: Maybe just store the model get and use `Model.path()`? This doesn't actually work for version prefixes. Same below.

File: bentoml/_internal/cli/bento_management.py (hunk 1 above)
Comment (larme) [reply bentoml/_internal/cli/bento_management.py]: I think output examples is a bit too long to be included here. But I made some improvements on the doc strings.

**Review by sauyon — APPROVED** [review]
LGTM 👍

**Pull request merged**